### PR TITLE
Updated bind_rows document.

### PR DESCRIPTION
**Pull request by hidekoji**
Updated the bind_rows document with the latest information.

- Data Frame Name is automatically added as a new ID column
- Added option "Force Data Type to Match"

### CODE REVIEW

File: wrangling/bind_rows.md [hunk 1]
```diff
@@ -6,11 +6,19 @@ Bind multiple data frames by row.
...
-1. Select Data Frames that you want to bind.
-2. Column Name for Data Frame ID - Data frames identifier. When .id is supplied, a new column of identifiers is created to link each row to its original data frame. The labels are taken from the named arguments to bind_rows(). When a list of data frames is supplied, the labels are taken from the names of the list. If no names are found a numeric sequence is used instead.
+1. Select Operation Type for the Merge. Supported options are:
+
+- Bind Rows (Merge all rows)
+- Union (Merge only unique rows)
+- Intersection (Merge only the common)
+- Setdiff (Merge only the difference)
+
+2. Select Data Frames that you want to bind.
```
Comment (kanaugust) [inline wrangling/bind_rows.md]: 2. Select the Data Frames you want to merge. You can select a single or multiple data frames.

File: wrangling/bind_rows.md [hunk 2]
```diff
@@ -6,11 +6,19 @@ Bind multiple data frames by row.
...
+3. Reserve Data Frame Name - When this checkbox is checked, Data Frame Name is automatically added as a new ID column.
```
Comment (kanaugust) [inline wrangling/bind_rows.md]: 3. (Optional) You can enable 'Reserve Data Frame Name' option to create a new column to hold the names of merged data frames.

File: wrangling/bind_rows.md [hunk 3]
```diff
@@ -6,11 +6,19 @@ Bind multiple data frames by row.
...
+4. Force Data Type to Match - When this checkbox is checked, it allows you to merge data frames whose column data types are different. After the merge, it detects and assigns appropriate data types to these columns.
```
Comment (kanaugust) [inline wrangling/bind_rows.md]: 4. (Optional) You can enable 'Force Data Type to Matc' to merge the data frames even when the data types of the corresponding columns are different. After the merge, it detects and assigns appropriate data types automatically.

File: wrangling/bind_rows.md (hunk 1 above)
Comment (hidekoji) [reply wrangling/bind_rows.md]: Updated as suggested

File: wrangling/bind_rows.md (hunk 2 above)
Comment (hidekoji) [reply wrangling/bind_rows.md]: Updated as suggested.

File: wrangling/bind_rows.md (hunk 3 above)
Comment (hidekoji) [reply wrangling/bind_rows.md]: Updated as suggested.

**Pull request merged**